app-config-electron/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import { WebPreferences } from 'electron';
+import { config } from '@app-config/main';
+import * as path from 'path';
+
+export function addAppConfigPreload(baseWebPreferences?: WebPreferences) {
+  let webPreferences: WebPreferences = {};
+
+  if (baseWebPreferences) {
+    webPreferences = baseWebPreferences;
+  }
+
+  const userPreload = baseWebPreferences?.preload;
+  const userArguments = baseWebPreferences?.additionalArguments;
+
+  const preloadArguments = userArguments || [];
+
+  preloadArguments.push(`--app-config=${JSON.stringify(config)}`);
+
+  if (userPreload) {
+    preloadArguments.push(`--user-preload=${userPreload}`);
```

Comment:
I assume that this workaround is because of `preload` being singular right?

Reply:
Yep, if a user wanted to use their own preload in addition to ours they couldn't without that